server/data/stock-market.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require("mongoose");
+const {stockMarketInfo} = require("../dal/stock-markets");
+
+const MARKET_DATA = [{
+    market_name: "Nasdaq",
+    stocks: [
+        {
+            ticker: "TSLA",
+            name: "Tesla"
+        },
+        {
+            ticker: "AMZN",
+            name: "Amazon"
+        },
+        {
+            ticker: 'GOOGL',
+            name: "Google"
+        },
+        {
+            ticker: 'FB',
+            name: "Facebook"
+        },
+        {
+            ticker: 'TWTR',
+            name: "Twitter"
+        },
+        {
+            ticker: 'UBER',
+            name: "Uber Technologies Inc"
```

Comment:
If you're planning on including "Inc" here rather than just "Uber" then I think you should be consistent and use it everywhere it's required. Or just use "Uber" rather than the full legal name of the companies. Just a minor consistency issue.